docs/1.concepts/data-flow/near-data-flow.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 1
+---
+
+# NEAR Data Flow
+
+## Video explanation
+
+In the video below we have tried to give a quick overview on main concept of how the data is flowing in NEAR Protocol blockchain. You will find a text version below the video.
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VSBJ-A69Km4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Text version
```

Comment:
We can remove the title and the explanation that the text and video are the same.

Reply:
removed